apps/fail2ban.sh

```diff
@@ -27,18 +27,28 @@ debug_mode
 root_check
 
 # Check if fail2ban is already installed
-if ! is_this_installed fail2ban
+if ! [ -f /etc/fail2ban/filter.d/nextcloud.conf ] || is_this_installed fail2ban
 then
     # Ask for installing
     install_popup "$SCRIPT_NAME"
 else
     # Ask for removal or reinstallation
     reinstall_remove_menu "$SCRIPT_NAME"
     # Removal
-    print_text_in_color "$ICyan" "Unbanning all currently blocked IPs..."
-    fail2ban-client unban --all
-    check_command apt-get purge fail2ban -y
-    rm -Rf /etc/fail2ban/   
+    if ! does_this_docker_exist bitwarden_rs
+    then
+        print_text_in_color "$ICyan" "Unbanning all currently blocked IPs..."
+        fail2ban-client unban --all
+        apt purge fail2ban -y
+        rm -rf /etc/fail2ban
+    else
+        print_text_in_color "$ICyan" "Unbanning all currently blocked IPs..."
+        fail2ban-client unban --all
```

Comment:
I guess this would also unban all bitwarden_rs banned ip addresses but should most likely be fine...